scripts/repair_batlas_references.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
```

Comment:
Python 2.7 supports `print()` without this, doesn't it?

Reply:
Looks like the parens are interpreted as a tuple:
```
('Input File:', 'var/full.json')
('Dry Run:', True)
```

Reply:
Oh right! Python2.7 `print()` isn't equivalent when used with more than one arg.